apps/participants/views.py

```diff
@@ -131,6 +131,14 @@ def participant_team_detail(request, pk):
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
     if request.method == "GET":
+        if not is_user_part_of_participant_team(
+            request.user, participant_team
+        ):
+            response_data = {
+                "error": "You are not a authorized to change team details!"
```

Comment:
`"You are not a authorized to change team details!"` -> `" Sorry, You are not authorized to view team details."`

Reply:
@Ram81 Can you please update this?